### DISCUSSION TITLE
Unhandled error when opening beatmap page

### DISCUSSION THREAD

**Discussion by Jeglerjeg** [ppy/osu#15417] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31293237/139912321-4fb94d93-1884-4576-8191-f4307cf90b15.png)

[database.log](https://github.com/ppy/osu/files/7462708/database.log)
[network.log](https://github.com/ppy/osu/files/7462709/network.log)
[performance.log](https://github.com/ppy/osu/files/7462710/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7462711/runtime.log)

**Comment by bdach** [comment, marked as answer]
I can't reproduce when opening via the beatmap listing.

How did you open the beatmap overlay there exactly? I'm interested in through which screens in particular.

**Reply by Jeglerjeg** [reply]
I opened it through the top100 section on my profile screen

**Reply by bdach** [reply]
hmm. right. i pieced the same together from `network.log`.

that said i repeated the exact same steps and it still doesn't fail...

**Reply by Jeglerjeg** [reply]
I can consistently reproduce it every time I open any beatmap from my top100 section at least, although opening one from song select doesn't trigger the exception

**Reply by Jeglerjeg** [reply]
Update: it happens from song select if it's a map I have a score on. Just loading the leaderboard in song select is enough to trigger the exception, opening the beatmap page also does so but it's not necessary.
New runtime log:
[runtime.log](https://github.com/ppy/osu/files/7462909/runtime.log)

**Reply by bdach** [reply]
this is the same as https://github.com/ppy/osu/discussions/15422, right?

**Reply by Jeglerjeg** [reply]
it looks similar, it might be the same. however I don't crash, and it doesn't throw an exception unless I have a score on the global leaderboard (my score doesn't show up either)

**Reply by bdach** [reply]
have checked and those are the same issue. going to mark this as resolved as that one has a fix linked.
